src/storage/mongodb/TagStorage.ts

```diff
@@ -39,6 +39,118 @@ export default class TagStorage {
     await Logging.traceEnd(tenantID, MODULE_NAME, 'saveTag', uniqueTimerID, tagMDB);
   }
 
+  public static async saveImportedTag(tenantID: string, importedTagToSave: ImportedTag): Promise<void> {
+    const uniqueTimerID = Logging.traceStart(tenantID, MODULE_NAME, 'saveImportedTag');
+    const tagMDB = {
+      _id: importedTagToSave.id,
+      description: importedTagToSave.description,
+      importedOn: new Date(),
+      importedBy: Utils.convertToObjectID(importedTagToSave.importedBy),
+      status: importedTagToSave.status,
+      error: importedTagToSave.error
+    };
+    await global.database.getCollection<any>(tenantID, 'tagsImport').findOneAndUpdate(
+      { _id: tagMDB._id },
+      { $set: tagMDB },
+      { upsert: true, returnOriginal: false }
+    );
+    // Debug
+    await Logging.traceEnd(tenantID, MODULE_NAME, 'saveImportedTag', uniqueTimerID, tagMDB);
```

Comment:
Should return the created tag ID for consistency reason like other API.

Reply:
Done